index.html

```diff
@@ -9,11 +9,105 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
+    <div class="box-area">
+        <!-- Nav Bar -->
+        <header>
+            <div class="wrapper">
+                <div>
+                    <img src="img/karma-logo.svg" alt="Karma Logo" class="header-logo">
+                </div>
+                <nav>
+                    <a href="#">Meet Karma</a>
+                    <a href="#">How it Works</a>
+                    <a href="#">Store</a>
+                    <a href="#">Blog</a>
+                    <a href="#">Help</a>
+                    <a href="#">Login</a>
+                </nav>
+            </div>
+        </header>
+        <main>
+            <div class="banner-area">
+                <div class="wrapper">
+                    <!-- First section -->
+                    <div class="img">
+                        <img src="img/first-background.jpg" alt="a person sitting at a table in a bar with a tablet and the karma WiFi device">
+                    </div>
+                    <div class="text-container">
+                        <h1>Introducing Karma</h1>
+                        <p>Bring WiFi with you, everywhere you go.</p>
+                        <div class="button">
+                            <button class="button1">Learn More</button>
+                        </div>
+                    </div>
+                </div>           
+            </div>
+            <!-- Second Section -->
+        <section> 
+                <div class="content-area">
+                    <div class="wrapper">
```

Comment:
Cosmetics: throughout the code, we want to keep the same styling. If you use 4 spaces or tab to indent inside, everywhere should follow the same. Here after `<section>` we only need 1 tab.